package.json

```diff
@@ -12,7 +12,7 @@
     "@babel/preset-env": "^7.4.4",
     "@babel/preset-react": "^7.0.0",
     "@babel/preset-typescript": "^7.3.3",
-    "@equinor/fusion": "^0.4.54",
```

Comment:
Should update the dependency to @equinor/fusion in the app template aswell (templates/app/package.json but don't use npm install there)

Reply:
Done